administrator/components/com_media/controllers/api.json.php

```diff
@@ -81,7 +81,8 @@ class MediaControllerApi extends Controller
 	public function files()
 	{
 		// Get the required variables
-		$path = $this->input->getPath('path', '/', 'path');
+		list($adapterInfo, $path) = explode(':', $this->input->getString('path', 'local:/'));
+		list($adapter, $account)  = array_merge(explode('-', $adapterInfo), array(""));
```

Comment:
This shows how multiple accounts can be integrated. 
Imo we should not include it until we really use it. Please remove the account handling.